### DISCUSSION TITLE
How is "My Blog" changed in the sample theme?

### DISCUSSION THREAD

**Discussion by faffr** [lxndrblz/anatole#436] [Q&A] [answered]
I cannot seem to locate where "My Blog" is specified in the "params.toml" file. The "config.toml" file does not contain it either. Apologies if this is a very rudimentary question. I am new to all of this. 
![myblog_pic](https://user-images.githubusercontent.com/31292398/227756010-207564eb-89da-409a-a993-27a047b4b6f0.jpg)

**Comment by lxndrblz** [comment, marked as answer]
@faffr If your using a bilingual setup the title will be overwritten from the `languages.toml` files:

https://github.com/lxndrblz/anatole/blob/923c738f9642fd3ff21c0463d70345ee62f52f3c/exampleSite/config/_default/languages.toml

If you don't have a bilingual setup you can define it in the `params.toml`:
https://github.com/lxndrblz/anatole/blob/923c738f9642fd3ff21c0463d70345ee62f52f3c/exampleSite/config/_default/params.toml

**Reply by faffr** [reply]
thank you!
